packages/common/src/content/_internal/ContentUiSchemaEdit.ts

```diff
@@ -0,0 +1,130 @@
+                type: "ERROR",
+                keyword: "required",
+                icon: true,
+                labelKey: "{{i18nScope}}.fields.title.requiredError",
+              },
+            ],
+          },
+        },
+        // summary
+        {
+          labelKey: "{{i18nScope}}.fields.summary.label",
+          scope: "/properties/summary",
+          type: "Control",
+          options: {
+            control: "hub-field-input-input",
+            type: "textarea",
+            helperText: {
+              labelKey: "{{i18nScope}}.fields.summary.hint",
+            },
+          },
+        },
+        // description
+        {
+          labelKey: "{{i18nScope}}.fields.description.label",
+          scope: "/properties/description",
+          type: "Control",
+          options: {
+            control: "hub-field-input-input",
+            type: "textarea",
+            helperText: {
+              labelKey: "{{i18nScope}}.fields.description.hint",
+            },
+          },
+        },
+        // featured image
+        {
+          labelKey: "{{i18nScope}}.fields.featuredImage.label",
+          scope: "/properties/view/properties/featuredImage",
+          type: "Control",
+          options: {
+            control: "hub-field-input-image-picker",
+            maxWidth: 727,
+            maxHeight: 484,
+            aspectRatio: 1.5,
+            helperText: {
+              labelKey: "{{i18nScope}}.fields.featuredImage.helperText",
+            },
+          },
+        },
+        // featured image alt text
+        {
+          labelKey: "{{i18nScope}}.fields.featuredImage.altText.label",
+          scope: "/properties/view/properties/featuredImageAltText",
+          type: "Control",
+          options: {
+            helperText: {
+              labelKey: "{{i18nScope}}.fields.featuredImage.altText.helperText",
+            },
+          },
+        },
+        // tags
+        {
+          labelKey: "{{i18nScope}}.fields.tags.label",
+          scope: "/properties/tags",
+          type: "Control",
+          options: {
+            control: "hub-field-input-combobox",
+            allowCustomValues: true,
+            selectionMode: "multiple",
+            placeholderIcon: "label",
+            helperText: { labelKey: "{{i18nScope}}.fields.tags.hint" },
+          },
+        },
+        // categories
+        {
+          labelKey: "{{i18nScope}}.fields.categories.label",
+          scope: "/properties/categories",
+          type: "Control",
+          options: {
+            control: "hub-field-input-combobox",
+            allowCustomValues: false,
+            selectionMode: "multiple",
+            placeholderIcon: "select-category",
+            helperText: {
+              labelKey: "{{i18nScope}}.fields.categories.agolHint", // TODO: hint should describe whether it can be set on Enterprise or Online
+            },
+          },
+        },
+      ],
+    },
+    // { // TODO: in another PR
```

Comment:
good to be forward thinking. I suspect that it will need to be added to `ContentSchema` as well?